source/triggers/disable.txt

```diff
@@ -103,3 +111,26 @@ importing an application directory with {+cli-bin+}.
 
                realm-cli push
 
+
+Restoring from a Snapshot
+-------------------------
+
+Consider the following scenario:
+
+1. A database trigger is disabled or suspended.
+
+#. New documents are added while the trigger is disabled.
+
+#. The database is restored from a snapshot to a time prior to the new documents 
+   being added.
+
+#. The database trigger is restarted.
+
+In this case, the trigger will pick up all of the new file additions and fire a 
+notification for each. It will not resend prior notifications. 
```

Comment:
I _think_ I get this but I'm not entirely sure. In my head this section kinda reads like "if you restore from a snapshot everything should just work" so I wanna make sure we're not missing something.

Would it be safe to say something like "the updates are applied on top of the snapshot and the trigger is able to pick up the change events for them"?